apps/web/components/PageWrapperAppDir.tsx

```diff
@@ -16,16 +13,6 @@ export type PageWrapperProps = Readonly<{
 }>;
 
 function PageWrapper(props: PageWrapperProps) {
-  const pathname = usePathname();
-  let pageStatus = "200";
-
-  if (pathname === "/404") {
-    pageStatus = "404";
-  } else if (pathname === "/500") {
-    pageStatus = "500";
-  } else if (pathname === "/403") {
-    pageStatus = "403";
-  }
```

Comment:
This detection wrongly sets status code to 200 if non-200 erros comes on some other paths